.github/workflows/ci.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-latest, macos-latest ]
-        python-version: [ 3.9 ]
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        python-version: [ "3.9" , "3.10.6" ]
```

Comment:
Do you really need to be that specific about the Python version here? What happens if you just use 3.10?

Reply:
For the moment it breaks because of `mypy` - see f3b5297 and https://github.com/python/mypy/issues/13627.